core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -68,6 +68,9 @@ private[spark] class Executor(
 
   // Make sure the local hostname we report matches the cluster scheduler's name for this host
   Utils.setCustomHostname(executorHostname)
+  
+  private val executorMetrics: ExecutorMetrics = new ExecutorMetrics
+  executorMetrics.setHostname(Utils.localHostName)
```

Comment:
I'll check that, if we cannot get the port here, we still can get that on Spark listener part (using `StorageStatusListener` or `JobProgressListener` as part of `MemoryListener`), I'll add `host:port` on the web anyway.